src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -213,6 +215,44 @@ protected final String getUriString(String filename) {
                 .toString();
     }
 
+    /**
+     * Returns the absolute system path of the given file.
+     *
+     * @param filename file name.
+     * @return absolute file path.
+     */
+    protected final File getInputFile(String filename) {
+        return new File("src/test/resources/" + getPackageLocation() + "/" + filename);
+    }
+
+    /**
+     * Performs verification of the config read from input file.
+     *
+     * @param testConfig hardcoded test config.
+     * @param parsedConfig parsed config from input file.
+     */
+    protected static void verifyConfig(DefaultConfiguration testConfig,
+                                      DefaultConfiguration parsedConfig) {
+        final String errorMessage = "Test configs and parsed configs differ.";
```

Comment:
lets use different error message for different cases

- `Check name {} differs from expected {}`
- `Property keys differ from expected`
- `Property value for key {} differs from expected`

or smth like that

Reply:
> ```Check name {} differs from expected {}```

Why do we need to display the name in the message. Truth already does that nicely. Please see: https://github.com/checkstyle/checkstyle/pull/10106#discussion_r651648922
I have changed the messages.

Reply:
okay